eng/pipelines/coreclr/templates/upload-jits-steps.yml

```diff
@@ -10,10 +10,15 @@ steps:
   displayName: Enable python venv
 
 # Ensure the Python azure-storage-blob package is installed before doing the upload.
-- script: $(PipScript) install --upgrade pip && $(PipScript) install azure.storage.blob==12.5.0 --force-reinstall
-  displayName: Upgrade Pip to latest and install azure-storage-blob Python package
+- script: $(PipScript) install --upgrade pip && $(PipScript) install azure.storage.blob==12.5.0 --force-reinstall && $(PipScript) install azure.identity==1.16.1 --force-reinstall
+  displayName: Upgrade Pip to latest and install azure-storage-blob and azure-identity Python packages
 
-- script: $(PythonScript) $(Build.SourcesDirectory)/src/coreclr/scripts/jitrollingbuild.py upload -build_type $(_BuildConfig) -arch $(archType) -host_os $(osGroup) -git_hash $(Build.SourceVersion) --use_latest_jit_change
-  displayName: Upload JIT to Azure Storage
-  env:
-    CLRJIT_AZ_KEY: $(clrjit_key1) # secret key stored as variable in pipeline
+- task: AzureCLI@2
+  displayName: 'Upload JIT to Azure Storage'
+  inputs:
+    azureSubscription: 'superpmi-collect-rw'
+    scriptType: 'pscore'
+    scriptLocation: 'inlineScript'
+    inlineScript: |
+      $Env:AZCOPY_AUTO_LOGIN_TYPE="AZCLI"
```

Comment:
Do you shell onto azcopy?

Reply:
@hoyosjs not sure I understand the question. we don't use `azcopy` here explicitly, but we use `AzureCliCredential` provider inside the python script we invoke here.
Perhaps, we can also remove it and use `DefaultAzureCredential` I didn't test it - is it an issue?

Reply:
It's not an issue - but `DefaultAzureCredential` should be enough. The only caveat BTW is the possibility that DefaultAzureCredential might pick up other ambient identities (MI's and Workload identities) before the Az CLI creds. if you ever hit this, just swap `DefaultAzureCredential` for `AzureCliCredential`

Reply:
Replaced `AzureCliCredential` with `DefaultAzureCredential`